src/Simulation/Simulation.jl

```diff
@@ -838,18 +840,24 @@ end
 ToDo...
 """
 function simulate!( sim::Simulation{T};  
-                    refinement_limits = [0.2, 0.1, 0.05 ],
+                    refinement_limits = [0.2, 0.1, 0.05],
```

Comment:
`simulate!()` only had one refinement as default. How does this compare to the old default as it is refined thrice now? Is the accuracy the same/better/worse?

Reply:
It is finer now.